nsone/rest/transport/basic.py

```diff
@@ -60,9 +60,9 @@ def handleProblem(code, resp, msg):
         # always pass the body to the handleProblem function
         try:
             resp = opener.open(request, timeout=self._timeout)
+            body = resp.read()
         except HTTPError as e:
             resp = e
-            body = resp.read()
```

Comment:
Thanks for the change, but `body = resp.read()` is still needed in this `HTTPError` handling clause. I think it should be:
```
        try:
            resp = opener.open(request, timeout=self._timeout)
            body = resp.read()
        except HTTPError as e:
            resp = e
            body = resp.read()
```